opentelemetry-instrumentation/tests/test_run.py

```diff
@@ -110,10 +109,6 @@ def test_execl(
 class TestArgs(TestCase):
     @patch("opentelemetry.instrumentation.auto_instrumentation.execl")
     def test_exporter(self, _):  # pylint: disable=no-self-use
-        with patch("sys.argv", ["instrument", "2"]):
-            auto_instrumentation.run()
-            self.assertIsNone(environ.get(OTEL_METRICS_EXPORTER))
```

Comment:
this test doesnt appear specifically tied to metrics, should it have been changed to check that OTEL_TRACE_EXPORTER was none?